### DISCUSSION TITLE
What is the least number of samples should we input in a JCAST program?

### DISCUSSION THREAD

**Discussion by Yanqiuye** [ed-lau/jcast#21] [Q&A]
Hi, I'm trying to learn JCAST, and I am wondering how many samples are supposed at lease for a JCAST practice. For example, when I'm running the rmats2 command, it suggests me to provide two groups of samples, it that mean I need to input RNA seq data which from two samples for the next steps?

**Comment by ed-lau** [comment]
Hi -- yes JCAST will require the output from rMATS, which gives the genome coordinates of splice junctions and whether they are consistently found across samples. The q value filters in JCAST allows removal of splice junctions that significantly differ across samples. You can run the pipeline with two replicates of the same sample to be input to rMATS.

**Comment by Yanqiuye** [comment]
Got it, thank you very much.

On Tue 17 Jan 2023 at 19:49, Edward Lau ***@***.***> wrote:

> Hi -- yes JCAST will require the output from rMATS, which gives the genome
> coordinates of splice junctions and whether they are consistently found
> across samples. The q value filters in JCAST allows removal of splice
> junctions that significantly differ across samples. You can run the
> pipeline with two replicates of the same sample to be input to rMATS.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ed-lau/jcast/discussions/21#discussioncomment-4710487>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5I6SEO7S4C4HAD5A2OHJGLWS3ZTHANCNFSM6AAAAAAT434MYE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

Yanqiu Ye

PhD Student

School of Biomolecular and Biomedical Science

University College Dublin

Belfield, Dublin 4
